src/components/views/messages/MLocationBody.tsx

```diff
@@ -115,11 +120,25 @@ export function LocationBodyContent(props: ILocationBodyContentProps) {
                 </div>
                 : null
         }
-        <div
-            id={props.bodyId}
-            onClick={props.onClick}
-            className="mx_MLocationBody_map"
-        />
+        {
+            props.tooltip
+                ? <TooltipTarget
+                    label={props.tooltip}
+                    alignment={Alignment.InnerBottom}
+                    maxParentWidth={450}
+                >
+                        <div
+                        id={props.bodyId}
+                        onClick={props.onClick}
+                        className="mx_MLocationBody_map"
+                    />
+                </TooltipTarget>
+                : <div
+                    id={props.bodyId}
+                    onClick={props.onClick}
+                    className="mx_MLocationBody_map"
+                />
```

Comment:
kinda feels like this ought to be a const set earlier and re-used in both branches of this ternary

Reply:
Fixed in 7d6e07c392